connectivity/FEATURE_BLE/source/cordio/source/PalGapImpl.cpp

```diff
@@ -1424,6 +1770,20 @@ ble_error_t PalGap::extended_create_connection(
     const uint16_t *maximum_connection_event_lengths
 )
 {
+#if MBED_CONF_CORDIO_TRACE_PAL_ECHOES
+    tr_info("Extended create connection - "
```

Comment:
Could you add traces below in `update_direct_advertising_parameters` ? There is error paths which are not covered.